examples/MvcWeb/Controllers/ProjectApproveController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.UnitOfWork;
+
+
+namespace MvcWeb.Controllers
+{
+    [Route("api/projects/approve")]
+    [ApiController]
+    public class ProjectApproveController : ControllerBase
+    {
+        public IUnitOfWork uow;
+
+        public ProjectApproveController(IUnitOfWork unitOfWork)
+        {
+            uow = unitOfWork;
+        }
+
+
+        public IEnumerable<Project> GetAllProjects()
+        {
+            return uow.Repository<Project>().Get().ToList();
+        }
+        [HttpPut("{id}")]
+        public IActionResult ApproveStatusId(int id)
+        {
+            var project = GetAllProjects().FirstOrDefault((p) => p.Id == id);
+            project.StatusId = 1;
```

Comment:
1. Create or use if exist ProjectStatus enum in Models folder
e.g. enum ProjectStatus { New = 0, Approved = 10, Rejected = 20}

2. Create void ChangeStatus(ProjectStatus status) method in the ProjectService  (Services folder).

3. Use the enum inside of that method like project.StatusId = (int)status;

4. Use this method in controller